test/unit/BondingManager.js

```diff
@@ -2372,6 +2372,35 @@ contract("BondingManager", accounts => {
         })
     })
 
+    describe("transcoderStatus", () => {
+        const transcoder = accounts[0]
+        const currentRound = 100
+
+        beforeEach(async () => {
+            await fixture.roundsManager.setMockBool(functionSig("currentRoundInitialized()"), true)
+            await fixture.roundsManager.setMockBool(functionSig("currentRoundLocked()"), false)
+            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound)
+
+            await bondingManager.bond(1000, transcoder, {from: transcoder})
+            await bondingManager.transcoder(5, 10, {from: transcoder})
+        })
+
+        describe("caller is not bonded to self", () => {
+            it("returns NotRegistered", async () => {
+                await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound + 1)
+                await bondingManager.unbond(1000, {from: transcoder})
+
+                assert.equal(await bondingManager.transcoderStatus(transcoder), TranscoderStatus.NotRegistered, "should return NotRegistered for caller not in transcoder pool")
```

Comment:
Let's update or remove the assert message since `transcoderStatus()` doesn't check if the address is in the transcoder pool.

Reply:
fixed in e6d92cf